soroban-env-common/src/env.rs

```diff
@@ -152,6 +152,15 @@ macro_rules! call_macro_with_all_host_functions {
                 /// running contract. Traps if the running contract was not
                 /// invoked by a contract.
                 {"4", fn get_current_contract() -> Object }
+
+                /// Return the protocol version of the current ledger as a u32.
+                {"5", fn get_ledger_version() -> RawVal }
```

Comment:
Is `ledger_version` the term we typically use for this? It makes it sound like a single ledger could exist in multiple states, like this is ledger number 10 version 1, then this is it in version 2. Should we just call it protocol version?

Maybe we should leave it out. From best I can tell Ethereum doesn't expose a version to contracts, at least not noted here: https://docs.soliditylang.org/en/latest/units-and-global-variables.html

@jonjove @tamirms @paulbellamy  Do you know if Ethereum exposes a version number to contracts so they can change their behavior depending on version?